### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by HarshitBagla**
Thank you for your pull request! Please read and

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request (replace the text as instructed)

- Replace this with your change description

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ﻿# New members please add your credentials to a random position in the file.
 # This is to remove conflicts with other people adding their credentials.
+- name: Harshit Bagla 
+  github: harshitbagla
+  image: HarshitBagla.jpg
+  facebook: harshit.bagla.5
+  twitter: harshitbagla
```
Comment (abishekvashok) [inline _data/students.yml]: Dont do carriage returns

File: _includes/popup.html [hunk 2]
```diff
@@ -3,8 +3,8 @@
     <div class="picture">
     {% if include.person.image %}
     {% if include.who == 'student' %}
-    <img src="images/students/{{include.person.image}}"
-    alt="{{include.person.name}}"
```
Comment (abishekvashok) [inline _includes/popup.html:7]: Dont add yourself statically into the index.html file make use of Jekyll. The current convention is to insert your name into the students.yml file and place your image in 240x240 pixels in the images/students folder.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please read the readme.md file and contributing.md file. Please follow the instructions in it and resolve the merge conflicts.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Fix the issues which @Abhi2424shek raised.

**Comment by robbyoconnor** [comment]
@HarshitBagla I'm failing to understand why people are removing and not reading the pull request template. In addition to this -- you don't even describe what you're doing accurately.

**Comment by HarshitBagla** [comment]
I have committed the changes on gh-pages

**Comment by robbyoconnor** [comment]
@HarshitBagla you have merge conflicts. Also I added back the pull request template which you deleted. Please read and check each of those checkboxes, as well as replace the text you are instructed to replace.

**Comment by robbyoconnor** [comment]
Okay @HarshitBagla -- at this stage -- you massively messed up. What the heck are you doing. I'm closing this. Look at how the site is set up.

**Comment by robbyoconnor** [comment]
I resolved the merge conflicts but you're changing files you shouldn't.

File: _data/students.yml [hunk 3]
```diff
@@ -1,562 +1,569 @@
...
-  github: benjaminguo
-  image: benjaminguo.jpg
-  twitter: guoJLbenjamin
-  bio: Cynical
-  lat: 1.3553794
-  lng: 103.86774439999999
+# New members please add your credentials to a random position in the file.
+# This is to remove conflicts with other people adding their credentials.
+
+- name: Prince John
```
Comment (abishekvashok) [inline _data/students.yml:4]: Can't understand why are you doing this again and again, please don't add carriage returns nor modify the entire page

File: _includes/popup.html [hunk 4]
```diff
@@ -3,8 +3,8 @@
...
+    <img src=""
+    alt=""
```
Comment (abishekvashok) [inline _includes/popup.html:7]: Please don't add empty tags please fix them as it was.

File: index.html [hunk 5]
```diff
@@ -181,62 +189,53 @@
...
                                     <i class="fa fa-code"></i>
                                     &nbsp;
+
                                     <span class="stu-name card-text" title="{{student.name}}">
                                         {{student.name}}
                                     </span>
                                 </p>
                             </div>
                         </div>
-                        {% include popup.html person=student who='student' %}
```
Comment (abishekvashok) [inline index.html:237]: Please don't remove these tags, they are needed for Jekyll.

File: index.html [hunk 6]
```diff
@@ -67,26 +67,34 @@
...
+</div>
+<br>
+
+<div style="text-align:center">
+  <span class="dot" onclick="currentSlide(1)"></span> 
+  <span class="dot" onclick="currentSlide(2)"></span> 
+  <span class="dot" onclick="currentSlide(3)"></span> 
+</div>
+
 <!-- End of Slider -->
```
Comment (robbyoconnor) [inline index.html:98]: Learn how Jekyll works. You're not even trying.

**Review by abishekvashok — COMMENTED** [review]
You have merge conflicts see the tutorials in contributing.md file and resolve them and do the needy fixes as stated. You are a good coder you can do it take it up as a challenge!

**Review by robbyoconnor — COMMENTED** [review]
@HarshitBagla, I'm not even sure what you did with most of this -- or why you did 99% of these changes. 

You claimed an add yourself to the site task -- just work on that. At this stage, I'd say trash your fork and re-fork and then fix things. 

Image needs to be `240 x 240 (pixels)` [w x h].

**_ONLY_** edit `_data/students.yml` and do not put carriage-returns in. 

Come on, I know you can do quality work!

**Comment by HarshitBagla** [comment]
How to undo all the changes in the index.html? (those student info)

**Comment by abishekvashok** [comment]
Just delete the fork and fork again @HarshitBagla

**Comment by robbyoconnor** [comment]
And DO NOT DELETE THE TEMPLATE!!

**Pull request closed without merging**